go-client/version_test.go

```diff
@@ -0,0 +1,52 @@
+package exasol_test_setup_abstraction_go
```

Comment:
A little documentation that this is a check for the project configuration would be nice.

Reply:
✅